test/mongodb.test.js

```diff
@@ -1072,6 +1072,122 @@ describe('mongodb connector', function () {
     });
   });
 
+  describe('replaceOrCreate', function() {
+    it('should create a model instance even if it already exists', function(done) {
+      Product.replaceOrCreate({name: 'newFoo'}, function(err, updatedProduct) {
+        if (err)
+          return done(err);
+        should.not.exist(updatedProduct._id);
+        should.exist(updatedProduct.id);
+        verifyData(updatedProduct.id);
+      });
+       function verifyData(id) {
```

Comment:
You spacing is a bit off here, looks like `function verifyData` has one more space in front?